### ISSUE TITLE
Dependencies error

### ISSUE THREAD

**Issue by stoppeye** [IchHabRecht/TYPO3-devtools#1]
Hi! I just downloaded the current version (0.2.1) of your extension from the TER and tried to install it, which resulted in an error: The extension "devtools" has invalid version constraints in depends section. Extension key is missing!

Checking ext_emconf.php revealed that there was an extra line with no data in the "depends" section:

```
  'constraints' => 
  array (
    'depends' => 
    array (
      'typo3' => '7.5.0-7.99.99',
      '' => '',
    ),
    'conflicts' => 
    array (
    ),
    'suggests' => 
    array (
    ),
  ),
```

Removing this line resolved the problem. Interestingly, this line isn't there, when I check the same file in your GIT repo. I don't know what happened between github and TER, but something seems to have gone wrong somewhere...

**Comment by IchHabRecht** [comment]
Hi,

Thank you for your report. Unfortunately this happened due to a misconfiguration of the TER due to extension upload. I fear I cannot do much here as it is not possible to re-upload an existing version.

**Comment by dwenzel** [comment]
@IchHabRecht is the trailing comma in 'depends' array leading to this error in TER? 
The issue also happens when loading via composer (require typo3-ter/devtools). 
And I couldn't find it on packagist.org nor load it via vcs (because there is no composer.json in the repo)
The only way to get it loaded is removing the invalid entry in local source code...

**Comment by IchHabRecht** [comment]
Hi,

as I said, this is no problem in the extension itself. This was messed up during the upload to TER and devtools is not the only extension having that issue. Currently there is no other way than removing the empty dependency which triggers the exception. I will prepare an updated version (as it seems to work in TYPO3 8 without any changes) in the next few days.

**Comment by stoppeye** [comment]
Ich bin ab dem 31.10.2016 wieder erreichbar. Ihre Mail wird nicht weitergeleitet. Bei dringenden Nachfragen wenden Sie sich bitte an info[at]eye.ch oder telefonisch an 061 416 91 81.

Mit freundlichen Gr�ssen
Michael Stopp

**Comment by IchHabRecht** [comment]
Should be solved with the latest 0.2.2 version.

**Comment by dwenzel** [comment]
Just tested with TYPO3 version 7.6.12 - works. Thank you!
